### DISCUSSION TITLE
leaveadvice.com: detection

### DISCUSSION THREAD

**Discussion by yashzade** [uBlockOrigin/uAssets#16790] [Q&A]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://leaveadvice.com/jackson-mississippi-workers-compensation-lawyers/`

### Category

detection

### Description

Ad blocker detected 

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Chromium Mobile: 107
filterset (summary): 
  network: 91888
  cosmetic: 68916
  scriptlet: 19154
  html: 0
listset (total-discarded, last-updated): 
  added: 
    fanboy-annoyance: 84075-35041, now
  default: 
    user-filters: 1-0, never
    adguard-mobile: 8805-31, 1m
    easylist: 61866-17041, now
    easyprivacy: 30975-1183, now
    plowe-0: 3680-987, now
    ublock-abuse: 79-0, now
    ublock-badware: 4347-122, now
    ublock-filters: 32804-423, now
    ublock-privacy: 276-8, now
    ublock-quick-fixes: 163-23, now
    ublock-unbreak: 1902-53, now
    urlhaus-1: 6076-0, now
filterset (user): [array of 1 redacted]
trustedset: 
  added: [array of 5 redacted]
  removed: 
    chrome-scheme
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 2134 ms (selfie)
  maxAssetCacheWait: 1404 ms
popupPanel: 
  blocked: 16
  blockedDetails: 
    leaveadvice.com: 8
    doubleclick.net: 1
    ezodn.com: 1
    ezoic.com: 1
    googletagmanager.com: 1
    imasdk.googleapis.com: 1
    jsdelivr.net: 1
    quantserve.com: 1
    vlitag.com: 1
```
</details>

**Comment by stephenhawk8054** [comment]
Is it a redirection from another link?

**Comment by yashzade** [comment]
Yes. 
They change the link after every click of a starting link

`https://olamovies.best/kung-fu-hustle-2004/`
this is staring link 1080p button

**Comment by stephenhawk8054** [comment]
Which country are you when accessing the site?

**Comment by MasterKia** [comment]
`https://olamovies.best/dl/file/52295`
=>
`https://link1s.com/wo354ola`
=>
`https://anhdep24.com/verifylv3/?wo354ola` => `https://anhdep24.com/`
(or `covemarkets.com`, `byboe.com`, `hocbeauty.com`, `nguyenvanbao.com`)
=>
`https://link1s.com/wo354ola`
=>
`https://check.olamovies.best/?id=2102444970`
=>
`https://new5.gdtot.cfd/file/2102444970`

***

`https://ez4short.com/hdy8mola`
=>
`https://techmody.io/safe2.php?link=hdy8mola`
=>
`https://ez4short.com/hdy8mola`

**Comment by MasterKia** [comment]
I get no `leaveadvice.com`.
This works:
```adb
olamovies.best##+js(nano-sib, counter)
anhdep24.com,hocbeauty.com,covemarkets.com,byboe.com,nguyenvanbao.com##.btn:has-text(Click):others()
anhdep24.com,hocbeauty.com,covemarkets.com,byboe.com,nguyenvanbao.com##+js(set, count, 0)

ez4mods.com,techmody.io##+js(acis, adBlockDetected)
ez4mods.com,techmody.io##+js(set, count, 0)
ez4mods.com,techmody.io##.btn#go_d:style(display: block !important;)
ez4mods.com,techmody.io##.btn#go_d:others()
```
